packages/dev/core/src/Cameras/targetCamera.ts

```diff
@@ -354,18 +359,58 @@ export class TargetCamera extends Camera {
 
         // Move
         if (needToMove) {
-            this._updatePosition();
+            if (this.inertia === 0) {
+                this._updatePosition();
+                this.cameraDirection.scaleInPlace(0);
+            } else {
+                this._updatePosition();
+
+                if (Math.abs(this.cameraDirection.x) < this.speed * Epsilon) {

```

Comment:
nit - `this.speed * Epsilon` can be calculated beforehand and be used here instead of calculating it 3 times.